sklearn/linear_model/ridge.py

```diff
@@ -123,14 +123,17 @@ def _solve_cholesky(X, y, alpha, sample_weight=None):
         return coefs
 
 
-def _solve_cholesky_kernel(K, y, alpha, sample_weight=None):
+def _solve_cholesky_kernel(K, y, alpha, sample_weight=1.0, copy=False):
```

Comment:
I think we had come to the conclusion that it was better to have `sample_weight=None` as default when there are no sample weights. I'll try to find the discussion.